Neos.ContentRepository.Core/Classes/Projection/Workspace/WorkspaceProjection.php

```diff
@@ -138,6 +138,7 @@ public function reset(): void
         $this->getDatabaseConnection()->exec('TRUNCATE ' . $this->tableName);
         $this->checkpointStorage->acquireLock();
         $this->checkpointStorage->updateAndReleaseLock(SequenceNumber::none());
+        $this->workspaceRuntimeCache->enableCache();
```

Comment:
on the other hand this will not really be called in production.

The same is with the `ContentSubgraphWithRuntimeCaches`. it seems we never reenable the cache if marked once as stale (during event handling)